webserver/src/activity-logger/generator/annotation-comment/annotation-generator.js

```
@@ -141,13 +141,14 @@
     verb = "added";
     verbURI = "http://activitystrea.ms/schema/1.0/add";
   }
+  let formattedType = formatActivityType(req.locals.annotation.type);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'formattedType' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)